common/lib/xmodule/xmodule/util/grading_policy.py

```diff
@@ -0,0 +1,38 @@
+"""
+Default grading policy for a course run.
```

Comment:
Should this, perhaps, go in ```edx-platform/common/lib/xmodule/xmodule/course_metadata_utils.py``` instead?

Reply:
That seems like a perfect place to put it, based on the present content of that file.